apps/meteor/client/.eslintrc.json

```diff
@@ -0,0 +1,257 @@
+{
+  "root": true,
```

Comment:
why do we still need a eslint config specific for client? why not merge it with the root one?

Reply:
My memory might be betraying me, but I think at the time some `eslint-react-plugin` rules conflicted with other ESLint rules (maybe due to ESLint version being outdated), as well as many of the configuration comes from the Fuselage monorepo. I wanted to review everything, but I thought this is a little bit out of the scope of this pull request.